ci/scripts/release.sh

```diff
@@ -15,8 +15,8 @@ gh release create "${BUILDKITE_TAG}" --generate-notes -d -p
 
 echo "--- Download artifacts"
 mkdir -p target/debug && cd target/debug
-buildkite-agent artifact download risingwave-release .
-mv risingwave-release risingwave
+buildkite-agent artifact download risingwave-ci-release .
```

Comment:
`ci-release` will include debug assertions and debug info. Is this expected for releases, to external users?

Reply:
I remembered that ci-release is building RisingWave in debug mode while others in release mode?

Reply:
> building RisingWave in debug mode while others in release mode

That's ci-dev. 😍

https://github.com/risingwavelabs/risingwave/blob/3e0d4d2873c47c1bd141ccd90d286c75fff06e6f/Cargo.toml#L61-L78

Reply:
We want debug info in our binary. So that if it crashes somewhere, it would be easier for us to read the stack trace.

Reply:
Okay. So I believe overflow checks are also necessary (an example: #5196). Not sure about debug assertions. 🤣

Reply:
Oh, we don't need debug assertions. So we need a new profile for release builds right? Or we just need to use the original "release" profile from cargo?

Reply:
Yes. The `release` profile is untouched.